packages/langium/src/workspace/document-builder.ts

```diff
@@ -14,15 +14,20 @@ import { interruptAndCheck, MaybePromise } from '../utils/promise-util';
 import { IndexManager } from '../workspace/index-manager';
 import { DocumentState, LangiumDocument, LangiumDocuments } from './documents';
 
+export interface BuildOptions {
+    validationChecks?: 'None' | 'All'
```

Comment:
Nitpick: I'm used to seeing lower-case configs, so `'none' | 'all'`, but that's a personal preference perhaps. Independent of how we deal with it in this case, we should be consistent about it.